target_chains/sui/cli/src/cli.ts

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env node
+
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import {
+  DefaultStore,
+  getDefaultDeploymentConfig,
+  SuiChain,
+  SuiContract,
+} from "contract_manager";
+import { PriceServiceConnection } from "@pythnetwork/price-service-client";
+import { execSync } from "child_process";
+import {
+  Connection,
+  Ed25519Keypair,
+  JsonRpcProvider,
+  RawSigner,
+} from "@mysten/sui.js";
+import { initPyth, publishPackage } from "./pyth_deploy";
+import { resolve } from "path";
+import {
+  buildForBytecodeAndDigest,
+  migratePyth,
+  upgradePyth,
+} from "./upgrade_pyth";
+
+const OPTIONS = {
+  "private-key": {
+    type: "string",
+    demandOption: true,
+    desc: "Private key to use to sign transaction",
+  },
+  contract: {
+    type: "string",
+    demandOption: true,
+    desc: "Contract to use for the command (e.g sui_testnet_0xe8c2ddcd5b10e8ed98e53b12fcf8f0f6fd9315f810ae61fa4001858851f21c88)",
+  },
+  path: {
+    type: "string",
+    default: "../../contracts",
+    desc: "Path to the sui contracts, will use ../../contracts by default",
+  },
+  endpoint: {
+    type: "string",
+    desc: "Price service endpoint to use, defaults to https://xc-mainnet.pyth.network for mainnet and https://xc-testnet.pyth.network for testnet",
+  },
+  "feed-id": {
```

Comment:
maye feed-ids? :think:

Reply:
I thought about using it like this:
`cli create-feed --feed-id f1 --feed-id f2`
Which makes more sense to use the singular word. Something like `docker run -p 80:80 -p 3000:3000`